ush/ioda/bufr2ioda/bufr2ioda_altkob_surface.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+import sys
+import numpy as np
+import numpy.ma as ma
+import os
+import argparse
+import math
+import calendar
+import time
+import copy
+from datetime import datetime
+import json
+from pyiodaconv import bufr
+from collections import namedtuple
+from pyioda import ioda_obs_space as ioda_ospace
+from wxflow import Logger
+
+
+def Compute_sequenceNumber(lon):
+
+    lon_u, seqNum = np.unique(lon, return_inverse=True)
+    seqNum = seqNum.astype(np.int64)
```

Comment:
Usually we only want time variables to be np.int64. Recommending you change to np.int32 unless @emilyhcliu thinks it's fine.

Reply:
Agreed and updated.